htlcswitch/link.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/go-errors/errors"
 	"github.com/lightningnetwork/lnd/channeldb"
 	"github.com/lightningnetwork/lnd/contractcourt"
+	"github.com/lightningnetwork/lnd/extpreimage"
```

Comment:
does this path work for testing? (I cant remember what our resolution was to the naming)

Reply:
Yes it does. When working on this fork, follow this guide: https://github.com/kinesis-exchange/lnd/wiki/Working-on-this-Fork

Reply:
@treygriffith killing the game Trey